server/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDecider.java

```diff
@@ -303,7 +303,7 @@ public Decision canAllocate(ShardRouting shardRouting, RoutingNode node, Routing
         assert shardSize >= 0 : shardSize;
         long freeBytesAfterShard = freeBytes - shardSize;
         if (freeBytesAfterShard < diskThresholdSettings.getFreeBytesThresholdHighStage(total).getBytes()) {
-            logger.warn(
+            logger.debug(
```

Comment:
This is also often emitted by desired balance computation to make it even more confusing.